src/Intersection.ts

```diff
@@ -81,6 +81,36 @@ export class Intersection {
     }
   }
 
+  /**
+   * Use the ray casting algorithm to determine if {@link point} is in the polygon defined by {@link points}
+   * @see https://en.wikipedia.org/wiki/Point_in_polygon
+   * @param point
+   * @param points polygon points
+   * @returns
+   */
+  static isPointInPolygon(point: Point, points: Point[]) {
+    const other = new Point(point).setX(
+      Math.min(point.x - 1, ...points.map((p) => p.x))
+    );
```

Comment:
```suggestion
    const other = new Point(-Number.MAX_VALUE, point.y);
```

Reply:
this is great, it can be also positive, no need for minus

Reply:
ok i ll commit this change then